### PR TITLE
FRW-8938 Adjusted for Data import from other sources.

### PR DESCRIPTION
**Pull request by olhalivitchuk**
## PR Description

https://spryker.atlassian.net/browse/FRW-8938

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/ca/dev/configure-data-import-from-an-s3-bucket.md [hunk 1]
```diff
@@ -657,4 +96,28 @@ $config[FileSystemConstants::FILESYSTEM_SERVICE]['s3-import'] = [
...
+     */
+    public function isDataImportFromOtherSourceEnabled(): bool
+    {
+        return true;
+    }
+}
+```
+
+
+
```
Comment (vitaliiivanovspryker) [inline docs/ca/dev/configure-data-import-from-an-s3-bucket.md:122]: Could you add, please, an example of data inmport configuration .yml file as we discussed?

Comment (olhalivitchuk) [reply docs/ca/dev/configure-data-import-from-an-s3-bucket.md:122]: Do you mean this part https://github.com/spryker/spryker-docs/pull/2787/files#diff-af56f3bb7b7cace2d291f88754064a2a9595d7ffc7dea180a20ef4e4d56561d5R38 ?

**Review by vitaliiivanovspryker — APPROVED** [review]

File: docs/ca/dev/configure-data-import-from-an-s3-bucket.md [hunk 2]
```diff
@@ -17,620 +17,59 @@ This document describes how to replace the original CsvReader with the one based
...
-        return $container;
-    }
-}
-```
-
-
-5. In `src/Pyz/Zed/DataImport/Business/DataImportBusinessFactory.php`, do the following:
+| NAME                      | VERSION |
+|---------------------------|---------|
+| DataImport                | ^1.25.0 |
```
Comment (fsmeier) [inline docs/ca/dev/configure-data-import-from-an-s3-bucket.md:24]: we should use the composer name - sometimes it is not obvious if you mean `spryker/my-module` or `spryker-shop/my-module`. Also its easier to copy&paste.

So suggestion is to use `spryker/data-import` and `spryker/flysystem-aws3v3-file-system`

Comment (olhalivitchuk) [reply docs/ca/dev/configure-data-import-from-an-s3-bucket.md]: Fixed

**Pull request merged**